planner/core/preprocess.go

```diff
@@ -453,6 +453,10 @@ func (p *preprocessor) checkBindGrammar(originNode, hintedNode ast.StmtNode, def
 			p.err = ddl.ErrOptOnTemporaryTable.GenWithStackByArgs("create binding")
 			return
 		}
+		tableInfo := tbl.Meta()
+		dbInfo, _ := p.ensureInfoSchema().SchemaByTable(tableInfo)
```

Comment:
Do we need to handle this error?

Reply:
The second return value is not an error. It means whether we can find the table. We have already checked this.